### ISSUE TITLE
ability to visualize trees alone

### ISSUE THREAD

**Issue by meren** [meren/anvio#325]
If the user only has a tree file, it should still be possible to visualize it on the interface. Currently we are asking for a data file and a tree file minimum.

**Comment by meren** [comment]
now this:

``` bash
cd tests/sandbox/files_for_manual_interactive
anvi-interactive -t tree.txt --manual -p PROFILE.db
```

produces this:

![image](https://cloud.githubusercontent.com/assets/197307/13056401/b0e6198c-d3da-11e5-990f-9cb6771caa31.png)

and of course everything else works (loading saving states, selecting bins, performing search, etc):

![image](https://cloud.githubusercontent.com/assets/197307/13056445/f4dffc70-d3da-11e5-8e6d-5aababe0c468.png)

yay for the versatility of the codebase.

**Comment by dcourtine** [comment]
Hi Meren, 

I searched all the issues to find the solution to a problem. This issue solved it.  
But, I tried to reproduce your code above, and I found a bug:
- It is not possible to display again the tree when a state `default` is present in the `PROFILE.db`. The interface is stuck in the *Loading...* step.

But it works fine when a saved *state*  has not the default name! 🎉 

My anvi'o codebase is:  
>Anvi'o version ...............................: esther (v6)
>Profile DB version ...........................: 31

To reproduce the bug:
I downloaded the tree from [iToL](https://itol.embl.de/itol.cgi), in Newick without internal node IDs.  
- `anvi-interactive -t test-tree.nwk -p tree-profile.db --manual`
- Save the profile as `default`
- Kick-off the interactive command
- `anvi-interactive -t test-tree.nwk -p tree-profile.db --manual`
- Endless *Loading...* 

I noticed the save behaviour when I only provide a data table with `-t`.

I am aware that this is a tiny bug. I just wanted there to be a record of this  little unforeseen event 😃

Best, 
Damien

**Comment by meren** [comment]
Hey @dcourtine, thank you very much for the update! 

But reading this I feel a bit lost. How did you solve it at the end? I am a bit lost :)

**Comment by ozcan** [comment]
@dcourtine we had an issue with states named `default` it is fixed now on `master` and `6.1`. Can you reproduce this problem if you change the name from `default` to something else?

**Comment by dcourtine** [comment]
Thank you @ozcan ! With the `master` (f6f28f38ef22ab2ab6af92e384fe06dbe3f99012) it works.

I hope you are doing well,
Damien
